src/Sentry.Profiling/Sentry.Profiling.csproj

```diff
@@ -2,10 +2,8 @@
 
   <PropertyGroup>
     <!-- TODO check and update the list of supported frameworks. -->
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;net6.0</TargetFrameworks>
```

Comment:
Out of curiosity - why do we need to add net8 here?

Reply:
The only 'strong' reason I can defend here is so it in this case is that if this package is added alone, it'll bring `Sentry` as a transient dep. If we don't have net8.0 here, it'll pull net6.0 from `Sentry`.

But wrt these decisions the mainstream idea is: In libs keep the lowest you need. But in reality there are some other cases where you want to bump (e.g ns2.0 is supported by net461 but causes issues due to rep resolution so years ago we had to ad net461 everywhere).

Recently on the .NET Discord someone from the .NET team was arguing against just saying "just keep the lowest" because we miss out on compiler improvements and other things added by the latest SDK when compiling to a newer target. Their take is to always support the latest version. Of course that has a cost so we can take that as a grain of salt.

This package specifically is something I hope we'll heavy invest, and will unblock new use cases for .NET folks using Sentry for performance monitoring. And since this is hopefully aligning with the core `Sentry` package eventually (as in, merging into a single page) having this already on net8.0 made sense to me.

Reply:
Thanks for the thorough explanation, makes sense.